### ISSUE TITLE
M> Get new logo production-ready for launch

### ISSUE THREAD

**Issue by kitblake** [rchain/bounties#237]
This is a MoSCoW breakdown of preparation in order to launch the new logo on the 15th of February during the Gov conf. This post is not complete and can be edited as we fine tune the lists.  
Some items require sub-lists that can posted, and edited, as messages in this issue. 
MoSCoW categories: Must have, Should have, Could have, Won't have.

**Must**
- Create logo files in key resolutions (need a list), for print and web (including favicon ico file), stored in Github (ref: https://github.com/rchain/style_guide_and_files)
- Create presentations for the conf with the new logo. All RChain presentations (new or old) must use the new logo. We need: a placeholder screen, a blank presentation with logo and RChain look&feel...
- The new website will be launched bearing the new logo. On other sites the logo needs to be updated. These include: http://rholang.rchain.coop, dev.rchain.coop?
- Create t-shirts for all participants. Need a design, digital files, locate a printer in Seattle, order size breakdown (XXL, XL, L, M, S).
- RHOC distribution cards (in progress).
- The scarlet and charcoal colors of the logo must be defined in RGB, Hex, CMYK, and PMS.

**Should**
- Update social media platforms with new logo: Twitter, Telegram groups, Facebook, YouTube, Reddit, Bitcointalk, Medium, QQ, Discord (need to collect a list of resolutions, e.g. Telegram 512x512px). Named asset files can be stored in Github. 
- Create a basic RChain Logo Style Guide (ref: https://github.com/rchain/style_guide_and_files/blob/master/rchain_style_guide_r1v2.pdf)
- So far the best idea for SWAG for the conf is webcam covers, very appropriate and practical.
- Banner ups

**Could**
- Business cards
- RChain ether.cards (bearing RHOCs, not Ether). See #150 

**Won't**
- Letterhead
- Booth design
- Handbills, handouts, flyers (need content and context)
- Note that the RHOC logo is not changing, so the token keeps the double R cattle brand on exchange and index sites.

Ref: #232

Measure of Completion:  new logo visible on all 'Must' items
Suggested budget: 9000 for 'Must' items, plus 5000 for 'Should items'

**Comment by kitblake** [comment]
**Social Media channel logo resolutions**
- Telegram 512x512px

**Comment by Keaycee** [comment]
@kitblake How about printing some handbills, Handout or souvenir for the Governance conference. Also having a pretty sizeable banners up a day before the event.

**Comment by AyAyRon-P** [comment]
What's the timeline on this? I can have a first round draft of the MUST and SHOULD items to you by EOD Sunday. To store the files in the PDF - style_guide_and_files.pdf where is the original artwork - INDD or PPTX? Do we have any go-by for the old presentations as a reference?

**Comment by ValeBF** [comment]
I have the social media Images - and I am working on the brand guidelines

**Comment by AyAyRon-P** [comment]
Here are the profile sizes for the the icons we need to use(my recommendation is black ground option): 
FACEBOOK: 170px X 170px
TWITTER:  400px X 400px
YOUTUBE: 800px X 800px
REDDIT: 256px X 256px 
TELEGRAM: 800px X 800px
BITCOINTALK: 80px X 80px
MEDIUM: 100px X 100px
DISCORD: 128px X 128px

RHOLANG.RCHAIN.COOP: 225px X 225px

**Comment by AyAyRon-P** [comment]
Can we discuss who's working on what before starting so we don't end up over lapping like we just did?

**Comment by AyAyRon-P** [comment]
I'll handle the first 4 bullets from the MUST segment.

**Comment by AyAyRon-P** [comment]
@ValeBF you're taking the STYLE GUIDE from the SHOULD, yes?

**Comment by AyAyRon-P** [comment]
@kitblake can you describe the RHOC distribution card?

**Comment by AyAyRon-P** [comment]
Talked through the project with Vale and here is where we are with dividing the work load:

**AARON**
* Create logo files in key resolutions (need a list), for print and web (including favicon ico file), stored in Github
* Create presentations for the conf with the new logo. All RChain presentations (new or old) must use the new logo. We need: a placeholder screen, a blank presentation with logo and RChain look&feel...
*The new website will be launched bearing the new logo. On other sites the logo needs to be updated. These include: http://rholang.rchain.coop, dev.rchain.coop?
*Create t-shirts for all participants. Need a design, digital files, locate a printer in Seattle, order size breakdown (XXL, XL, L, M, S).
*Business Cards
*Banner Ups


**VALERIA**
*Update social media platforms with new logo: Twitter, Telegram groups, Facebook, YouTube, Reddit, Bitcointalk, Medium, QQ, Discord (need to collect a list resolutions, e.g. Telegram 512x512px).
*Create a basic RChain Logo Style Guide (ref: https://github.com/rchain/style_guide_and_files/blob/master/rchain_style_guide_r1v2.pdf)
*Business Cards
*Get your RHOCs off cards
*RHOC distribution cards?

**Comment by ValeBF** [comment]
@kitblake Could you please make clear to me what are the *Get your RHOCs off cards *RHOC distribution cards?

**Comment by kitblake** [comment]
Sorry @ValeBF that was a joke. I was suggesting we use the "Get your RHOCs off" text on the ether.card. In the sense that the recipient should move the RHOCs out of the wallet. But it wasn't a serious suggestion.

**Comment by kitblake** [comment]
For the style guide we want to have the scarlet and charcoal colors of the logo defined in RGB, Hex, CMYK, and PMS (added to the Musts in the MoSCoW above).

The previous guide defines the scarlet as RGB 204/30/70, CMYK 13/100/70/4, Hex #cc1e46. No PMS is provided. 
The Pantone is tricky, @AyAyRon-P will check a physical swatch book. It could be the 186 C red (using RGB for conversion) or (using CMYK) "Strong Red C" but 193 C or 1935 C look better on screen. This is why we need to look at the physical, printed color. :) 

The charcoal CMYK values are 82/70/56/62.

**Comment by kitblake** [comment]
@ValeBF @AyAyRon-P I've created a folder for the style guide and assets at:
https://github.com/rchain/Members/tree/master/style_guide_assets

When all is said and done we might move the folder to another repository, but here we can work and upload files to review progress.

**Comment by AyAyRon-P** [comment]
I working out the HEX and PMS tones now. 

Also, for the webcam covers were we thinking hardcovers or sticker covers?

**Comment by kitblake** [comment]
Ping @patrick727 on that one. 
Here's a hard cover that seems like it'll work on various kinds of laptops: 
https://www.motivators.com/Promotional-Custom-PrivacyCoverwithScreenCleanerFullColor-155109.html

**Comment by ValeBF** [comment]
And about the "must list"

I have been working on the new website whit the developer (These include: http://rholang.rchain.coop) we are in progress

**Comment by kitblake** [comment]
About subtlety in the new logo (as @AyAyRon-P mentioned a while back in Discord). One of the early mockups had multiple colors in the dot grid. That was visually attractive and would work well in RGB environments. It could also fall back to two colors when needed. The logo evolution eventually went for two colors, scarlet and charcoal. But to play the devil's advocate, I hacked the early mockup to remove the green tint, both desaturating and slightly lightening the green dots. This is the result:
![logo 3desaturated](https://user-images.githubusercontent.com/912545/35198368-2712d54c-feee-11e7-92ea-de89fa3733ef.png)

**Comment by Ojimadu** [comment]
I like the contrast in the one above. But am wondering was there any mockup that had green colour? I probably missed that.

**Comment by AyAyRon-P** [comment]
@kitblake I've already built all of the logos as 2 color - JPG, PNG, EPS, AI, and PSD in both CMYK and RGB. I don't think at this time a 3 color flat is necessary but what I would like to do is a 4C gradient option. Here is a very rough mockup to give the idea. 
![rchain_fnl_3d_whitebg_rgb](https://user-images.githubusercontent.com/35570702/35205565-cb3d5392-fefb-11e7-9208-3d295bd6ee51.jpg)

**Comment by ValeBF** [comment]
Personally, I prefer the flat design

**Comment by ICA3DaR5** [comment]
Me too!

**Comment by Keaycee** [comment]
The Logo presented by @AyAyRon-P is nice. It does has a 3D visual effect. @ValeBF I also prefer the flat design shown by @kitblake . The flat design would better on our new Ether. card.

**Comment by AyAyRon-P** [comment]
There are various applications for both, one isn't simply better than the other. The 4-C gradient option would be for very high level items(+/-10%) where visibility/resolution wouldn't be an issue and the availability for higher detail were there. The 2D icon would be the primary and used in probably close to 80% while there will be the remaining +/- 10% in black and white.

**Comment by kitblake** [comment]
I prefer the flat logo as well. The gradient is an interesting experiment though. It resembles Apple's former skeuomorphism. The dark dots look like iron rivets. The red ones sorta like matte droplets.

**Comment by Viraculous** [comment]
I think better of with the flat logo but what if the logos are placed in an ash(light-grey) background than white?

**Comment by AyAyRon-P** [comment]
@Viraculous there is no background color. We've opted not to frame out the icon with a background because a frame can create more issues for print and production than it's intended value could add.  The icon sets we've developed are for white/light color backgrounds or black/dark color backgrounds. These are the lowest denominators for production and the reason why we've developed them this way.

**Comment by patrick727** [comment]
Hello All, 

This work is extremely cool and i'm very grateful for your willingness to work on this task. 

I spoke with @kitblake about this specifically,...but there has been a change in plans. 

The logo update will need to be put on hault as it is going to be used as an example in the governance forum of decision making process.

**Comment by Viraculous** [comment]
OK. Its all nice anyway. The ash network background of the flat logo can still serve @AyAyRon-P .

**Comment by kitblake** [comment]
We've archived the work that was done for this issue. After getting the go people were scrambling to get everything ready to launch. The results are top notch!

Now that the new logo has been put on hold, some things, like the style guide, remain unfinished. Following is a status listing.

**Print**
https://github.com/rchain/Members/tree/master/style_guide_assets/print
- RChain T-shirt designs, 3 variants (mockups, print files are ready)
- Webcam Cover mockup (SWAG for the Gov Conf, production ready)
- Design for the rhoc.card (ether.card, print file ready)
- Banner Up variants, to use at conferences
- Mockups for business cards

Files are available in AI, EPS, and/or PSD.

**Web**
https://github.com/rchain/Members/tree/master/style_guide_assets/web
- RChain logo, transparent, for light backgrounds
- RChain logo, transparent, for dark backgrounds

All files are in web resolution or higher.
All files are available in 2 colors, 1 color, and B&W.
File formats include PSD, AI, EPS, PNG, JPG, PSD, and ICO (true ICO files).

**Presentations**
https://github.com/rchain/Members/tree/master/style_guide_assets/presentations
- Powerpoint templates, three options
- PDF templates, three options

File formats: POTX, PDF

**Social Media**
https://github.com/rchain/Members/tree/master/style_guide_assets/social_media  
This folder includes various assets for all the current social media channels:
- Facebook
- LinkedIn
- Medium
- Reddit
- Telegram
- Twitter
- YouTube

Files are in various resolutions and formats, following specs for each channel.

**RChain Style Guide**
https://github.com/rchain/Members/tree/master/style_guide_assets/RCHAIN_Style_Guide
- Style Guide PDF
- Email signature, 3 variants

The Style Guide is unfinished. It's mainly missing key textual content, and it also hasn't yet been thoroughly reviewed by stakeholders.

Kudos to @ValeBF and @AyAyRon-P for the top quality work and professional execution!
Also thanks to anyone else who contributed to this issue, and to all those who participated in the larger effort.

**Comment by lapin7** [comment]
@AyAyRon-P please send your email address to ops@rchain.coop
and join
https://groups.google.com/forum/#!managemembers/rchain/join

**Comment by AyAyRon-P** [comment]
@lapin7 Email sent

**Comment by AyAyRon-P** [comment]
Group joined!

**Comment by kitblake** [comment]
@pmoorman Closing.
